Realm/Realm/Realm.cs

```diff
@@ -1323,10 +1322,12 @@ public void RemoveAll()
         /// non-null <see cref="RealmConfigurationBase.EncryptionKey"/>, the copy will be encrypted with that key.
         /// </summary>
         /// <remarks>
-        /// The destination file cannot already exist.
-        /// <para/>
-        /// If this is called from within a transaction it writes the current data, and not the data as it was when
-        /// the last transaction was committed.
+        /// 1. The destination file cannot already exist.
+        /// 2. For synchronized realms, the file written will have the client file identifier removed.
```

Comment:
I don't think we need to explicitly mention this - the client file identifier is an implementation detail and not something exposed to users, so the fact that we're removing it is kind of irrelevant.

Reply:
Removed with 192bcec.